### PR TITLE
langchain, community: Implement Ontotext GraphDB QA Chain

### PR DESCRIPTION
**Pull request by nelly-hateva**
- **Description:** Implement Ontotext GraphDB QA Chain
  - **Issue:** N/A
  - **Dependencies:** N/A
  - **Twitter handle:** @OntotextGraphDB

### CODE REVIEW

**Comment by nelly-hateva** [comment]
I've synced this branch with the master branch. @baskaryan, @eyurtsev, @hwchase17, please, approve the workflows again and provide feedback. Is there anything else I should do, so this PR is merged?

**Review by baskaryan — COMMENTED** [review]
sorry for delay @nelly-hateva!

35k lines is pretty big, could we reduce the size of the various example files?

also would it be clearer to name classes OntoTextGraphDB... instead of GraphDB...? GraphDB sounds like an abstraction more than a specific integration, may confuse folks

**Comment by nelly-hateva** [comment]
Hello @baskaryan . Thank you for the quick reply. I completely agree with you and I've deleted all example files and heavily reduced the size of the changes. I've also renamed classes containing "GraphDB" to "OntotextGraphDB" (note that you proposed "OntoTextGraphDB", but "Ontotext" is a company name and it's written with lowercase "t") and added links to the website for clarity in the classes docs. Please, approve the workflows.

File: docs/vercel.json [hunk 1]
```diff
@@ -1444,6 +1444,10 @@
       "source": "/en/latest/modules/chains/examples/graph_cypher_qa.html",
       "destination": "/docs/use_cases/graph/graph_cypher_qa"
     },
+    {
+      "source": "/en/latest/modules/chains/examples/graph_ontotext_graphdb_qa.html",
+      "destination": "/docs/use_cases/graph/graph_ontotext_graphdb_qa"
+    },
```
Comment (baskaryan) [inline docs/vercel.json:1450]: only needed for when we've moved doc pages around and wanted to not break old links. no need to do this for new docs

File: libs/langchain/langchain/graphs/ontotext_graphdb_graph.py
Comment (baskaryan) [inline libs/langchain/langchain/graphs/ontotext_graphdb_graph.py:1]: no need to add this, only needed for backwards compatibility for classes that used to live in libs/langchain and were moved

File: docs/vercel.json (hunk 1 above)
Comment (nelly-hateva) [reply docs/vercel.json]: Tnx. Fixed

File: libs/langchain/langchain/graphs/ontotext_graphdb_graph.py
Comment (nelly-hateva) [reply libs/langchain/langchain/graphs/ontotext_graphdb_graph.py:1]: Tnx. Removed. Do I need the changes in libs/langchain/langchain/graphs/__init__.py and in libs/langchain/tests/unit_tests/graphs/test_imports.py or not?

File: libs/langchain/langchain/graphs/__init__.py [hunk 2]
```diff
@@ -35,4 +35,5 @@ def __getattr__(name: str) -> Any:
     "RdfGraph",
     "ArangoGraph",
     "FalkorDBGraph",
+    "OntotextGraphDBGraph",
```
Comment (baskaryan) [inline libs/langchain/langchain/graphs/__init__.py:38]: delete

Comment (nelly-hateva) [reply libs/langchain/langchain/graphs/__init__.py]: Thank you. Removed

**Comment by baskaryan** [comment]
awesome stuff, thank you @nelly-hateva!!

**Pull request merged**